src/app/components/core/notifications/add-notification-wizard/add-notification-wizard.component.html

```diff
@@ -210,6 +208,11 @@
                   </div>
                 </div>
               </div>
+              <div
+                class="icon is-small is-tooltip-right tooltip is-pulled-right is-hovered help-icon notification-type-help"
+                data-tooltip="Help" (click)="goToNotificationTypeLink()">
+                <i class="far fa-question-circle"></i>
+              </div>
```

Comment:
For Retrigger Time description "A dead period (in seconds), during which notifications will not be sent regardless of the notification type."

see more around here. We may want to add same info in the doc page linked here.

Reply:
Right now for Retrigger time description we are showing `Retrigger time in seconds for sending a new notification` on tooltip. Do you mean to change that into `A dead period (in seconds), during which notifications will not be sent regardless of the notification type.`?

Reply:
Yes